proto/sourcehub/orbis/tx.proto

```
@@ -44,25 +45,33 @@ message MsgUpdateParamsResponse {}
+  optional uint64 pss_interval = 4;
+  string policy_id = 5;
+  // Absent means no nonce. Provide a nonce to disambiguate rings with identical settings.
+  optional string nonce = 6;
 }
 
 message MsgCreateRingResponse {
   string ring_id = 1;
 }
 
+message MsgFinalizeRing {
+  option (cosmos.msg.v1.signer) = "creator";
```

Buf (1.69.0)

[error] cannot find `cosmos.msg.v1.signer` in this scope

(COMPILE)